app/src/main/java/bankingapp/CreateAccount.java

```diff
@@ -149,7 +100,7 @@ public static boolean luhnsAlgorithmCheck(String accountNumber){
      * @return String
      */
     public String toString(){
-        return "Thank you "+ getFirstName() + " " + getLastName() +"\nYour account has been created" +
+        return "Thank you "+ this.firstName + " " + this.lastName +"\nYour account has been created" +
```

Comment:
This is UI work. Generally, the `toString` method on a class like this is going to be for debugging convenience to the programmer (not the end user), or might be necessary to integrate with some third-party library, conceivably.

The message printed to the user should be controlled by the UI layer (which is currently the App class in this project), so you will probably need to keep the getter methods you had.